app/resources/api/v2/tube_from_tube_creation_resource.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    # @todo This documentation does not yet include a detailed description of what this resource represents.
+    # @todo This documentation does not yet include detailed descriptions for relationships, attributes and filters.
+    # @todo This documentation does not yet include any example usage of the API via cURL or similar.
+    #
+    # @note This resource cannot be modified after creation: its endpoint will not accept `PATCH` requests.
+    # @note Access this resource via the `/api/v2/tube_from_tube_creation/` endpoint.
+    #
+    # Provides a JSON:API representation of {TubeFromTubeCreation}.
+    #
+    # For more information about JSON:API see the [JSON:API Specifications](https://jsonapi.org/format/)
+    # or look at the [JSONAPI::Resources](http://jsonapi-resources.com/) package for Sequencescape's implementation
+    # of the JSON:API standard.
+    class TubeFromTubeCreationResource < BaseResource
+      ###
+      # Attributes
+      ###
+
+      # @!attribute [w] child_purpose_uuid
+      #   This is declared for convenience where child purpose is not available to set as a relationship.
+      #   Setting this attribute alongside the `child_purpose` relationship will prefer the relationship value.
+      #   @deprecated Use the `child_purpose` relationship instead.
```

Comment:
Is this deprecated comment correct? Same in below attributes. If this is a new resource and there is a preferred method of using relationships can we do that instead?

Assuming this would be riskier and mean changing limber requests?

Reply:
Yeah this is an approach I've been using because Limber works with UUIDs.  If I didn't add these "legacy" ways of passing in the relationships, I'd have to either pass the relationships around Limber (more change than I should be doing right now) or I'd have to call another endpoint to find the related object, just to be able to pass it to this endpoint.

Reply:
Ah ok, yeah I guess you have to limit the work somewhere. Is the plan to address these legacy methods after v2 is complete?